ReactWindows/ReactNative.Net46/Views/Text/ReactTextViewManager.cs

```diff
@@ -0,0 +1,171 @@
+﻿using ReactNative.UIManager;
+using ReactNative.UIManager.Annotations;
+using System.Collections;
+using System.Linq;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Documents;
+using System.Windows.Media;
+
+namespace ReactNative.Views.Text
+{
+    /// <summary>
+    /// The view manager for text views.
+    /// </summary>
+    public class ReactTextViewManager : ViewParentManager<RichTextBox, ReactTextShadowNode>
```

Comment:
Close, from what I can tell. The UWP one mixes in some extra functionality and automatically manages the FlowDocument, where the WPF one is more manual.